### ISSUE TITLE
rusage.contents could contain multiple equals signs

### ISSUE THREAD

**Issue by tomgreen66** [pygridtools/drmaa-python#22]
In helpers.py:
````python
    def adapt_rusage(rusage)
````

It splits the attributes in rusage.contents with:
````python
   k, v = attr.split('=')
````
On my system (PBS Pro 12.2 and pbs-drmma library) there is the possibility of an attribute containing an equals signs so it requires:
````python
    k, v = attr.split('=',1)
````

Otherwise it does not cleanly end the job and causes problems in software such as Galaxy.
Happy to provide patch.

**Comment by dan-blanchard** [comment]
I'd gladly accept a PR for that fix.  Thanks for pointing that out.  I've never used PBS, so I wasn't aware of that one.

**Comment by dan-blanchard** [comment]
Fixed by #23
